sportsref/nfl/players.py

```diff
@@ -177,6 +177,15 @@ def college(self):
         return college
 
     @sportsref.decorators.memoize
+    def collegeid(self):
```

Comment:
the other college function returns the college. this returns the player's college id so you could cross-reference their college page.